datafusion/core/tests/sql/order.rs

```diff
@@ -25,7 +25,7 @@ use test_utils::{batches_to_vec, partitions_to_sorted_vec};
 #[tokio::test]
 async fn sort_with_lots_of_repetition_values() -> Result<()> {
     let ctx = SessionContext::new();
-    let filename = "tests/parquet/data/repeat_much.snappy.parquet";
```

Comment:
the parquet one was particularly confusing because `tests/parquet` also has a bunch of rust code as well